app/models/concerns/file_set/geo_file_format_behavior.rb

```diff
@@ -38,20 +30,23 @@ def vector_work?
   end
 
   module ClassMethods
-    def image_file_formats
-      ['TIFF', 'IMAGE_FILE']
-    end
-
-    def raster_file_formats
-      ['TIFF_GeoTIFF', 'RASTER_FILE']
+    def gdal_formats
+      [
+        'image/tiff; gdal-format=GTiff',
+        'text/plain; gdal-format=AIGrid'
+      ].freeze
     end
 
-    def vector_file_formats
-      ['SHAPEFILE', 'SHAPEFILE_ZIP', 'VECTOR_FILE']
+    def ogr_formats
+      [
+        'application/zip; ogr-format=ESRI_Shapefile',
```

Comment:
Yeah. The quotes are problematic in the mime type. If you use single quotes, Fedora barfs. If you use double quotes, rails does funny stuff to the input. For example, rails removes the double quotes from `image/tiff; gdal-format="GTiff"`, and returns `image/tiff; gdal-format=GTiff`. However `application/zip; ogr-format="ESRI Shapefile"` is returned as is. I'll poke around a bit more and see if I can figure that out.